kapacitor/client.go

```diff
@@ -319,6 +319,9 @@ func (c *Client) Update(ctx context.Context, href string, rule chronograf.AlertR
 	} else {
 		opt, err = c.updateFromTick(rule)
 	}
+	if err != nil {
```

Comment:
How hard would it be to write a test that panics without this change and doesn't panic with it?

Reply:
Fixed in 4f77104 
In working through the tests, I found lots more possible issues.